examples/compute/v1beta1/networkfirewallpolicyrule.yaml

```
@@ -0,0 +1,121 @@
+apiVersion: compute.gcp.upbound.io/v1beta1
+kind: NetworkFirewallPolicyRule
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/networkfirewallpolicyrule
+  labels:
+    testing.upbound.io/example-name: primary
+  name: primary
+spec:
+  forProvider:
+    action: allow
+    description: This is a simple rule description
+    direction: INGRESS
+    disabled: false
+    enableLogging: true
+    firewallPolicySelector:
+      matchLabels:
+        testing.upbound.io/example-name: basic_network_firewall_policy
+    match:
+    - layer4Configs:
```

Uptest failed due to following error:
```
case.go:363: NetworkFirewallPolicyRule.compute.gcp.upbound.io "primary" is invalid: [spec.forProvider.match: Invalid value: "array": spec.forProvider.match in body must be of type object: "array", <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation]
```
You can try the following configuration:

```suggestion
    match:
      layer4Configs:
```